### ISSUE TITLE
gltfExporter causing memory leak

### ISSUE THREAD

**Issue by chwan1** [vrm-c/UniVRM#1960]
## Environments (please complete the following information):

 - UniVRM version: `0.107.0`
 - Unity version: `Unity-2021.3`
 - OS: `Windows 11`

## Describe the bug
I'm using the following code to export Glb and VRM at runtime. And after each execution the player memory increase by around 100MB. Most of the increased memory is used by Texture2D. The app is for exporting lots of Glb and VRM, so it's quite troubling.

Or is there any method to cleanup after Export that I'm missing?

```
GameObject exportRoot;
string GlbPath;

var data = new ExportingGltfData();
using (var exporter = new gltfExporter(data, new UniGLTF.GltfExportSettings(), null, null))
{
    exporter.Prepare(exportRoot);
    exporter.Export(new VRMShaders.RuntimeTextureSerializer());
}

var bytes = data.ToGlbBytes();
System.IO.File.WriteAllBytes(GlbPath, bytes);
```

**Comment by chwan1** [comment]
After investigating the code, I find that memory leak is caused by Texture2D is copied but never get destroied in https://github.com/vrm-c/UniVRM/blob/master/Assets/UniGLTF/Runtime/UniGLTF/IO/gltfExporter.cs#L331 

Chaning the code to following works for me
```
var exported = _textureExporter.Export();
for (var exportedTextureIdx = 0; exportedTextureIdx < exported.Count; ++exportedTextureIdx)
{
    var (unityTexture, colorSpace) = exported[exportedTextureIdx];
    GltfTextureExporter.PushGltfTexture(_data, unityTexture, colorSpace, textureSerializer);
    if (Application.isEditor)
    {
        GameObject.DestroyImmediate(unityTexture);
    }
    else
    {
        GameObject.Destroy(unityTexture);
    }
}
```

**Comment by ousttrue** [comment]
Thank you for reporting.
Check the Exporter's resource management and create a Dispose system if necessary.

**Comment by ousttrue** [comment]
![leak](https://user-images.githubusercontent.com/68057/214525495-6b0f6731-e851-4eda-907c-9af7c0ccef80.jpg)

**Comment by ousttrue** [comment]
Implemented disposal of Texture resources in #1960.
